Keshavk Gallery component with slider created

ksvkabra opened the pull request.

ksvkabra committed:
Gallery component with slider created

amritsrivastava commented:
@ksvkabra expand the timeline window of the gallery to its full width.
Also, add the functionality of autoplay in the gallery, the images are only moving through navigation buttons, it must change pictures after a delay of 3 - 5 sec.

ksvkabra committed:
`autoplay` added in gallery

ksvkabra committed:
image maxHeight updated and width of card updated

amritsrivastava committed:
Ksvkabra gallery (#4)

* feat: create sponsors page

* fix contact us page

* fix: imports to prevent build fail

* feat: WIP (International Press Application Form)

* add form submit link and fix css

* feat: create international press application form

* feat : Modified forms

* Link spreadsheet in IP Applicaiton form

* corrections: IP applicaiton form

* fix: styling of banner heading

* add metadata

* Modified Sponsors

* feat: Modify Sponsors Page

* Delete footer.js

* Delete contact.js

* update: UNSC country matrix

* update: country matrix

* feat: Modify Committees Page (#70)

* feat: Modify Committees Page

* Modified Committees Data

Co-authored-by: Amrit Srivastava <srivastava.amrit1999@gmail.com>

* Header registration dropdown created (#67)

* Added dropdown for registrations on navbar

* Added gallery route in header

* Updated registrations and about hover list data

* Added ClickAwayListener.
Added dropdown for about and registrations

Co-authored-by: Amrit Srivastava <srivastava.amrit1999@gmail.com>

* Blogs page created (#68)

* Blogs page added. Link for blog page added in header

* Created separate pages for blogs

* Added route for home blog in navbar and mobildrawer

* Ksvkabra keshavk blogs page (#2)

* feat: create sponsors page

* feat: WIP (International Press Application Form)

* feat: create international press application form

* feat : Modified forms

* Link spreadsheet in IP Applicaiton form

* corrections: IP applicaiton form

* fix: styling of banner heading

* add metadata

* Modified Sponsors

* feat: Modify Sponsors Page

* Delete footer.js

* Delete contact.js

* update: UNSC country matrix

* update: country matrix

* feat: Modify Committees Page (#70)

* feat: Modify Committees Page

* Modified Committees Data

Co-authored-by: Amrit Srivastava <srivastava.amrit1999@gmail.com>

* Header registration dropdown created (#67)

* Added dropdown for registrations on navbar

* Added gallery route in header

* Updated registrations and about hover list data

* Added ClickAwayListener.
Added dropdown for about and registrations

Co-authored-by: Amrit Srivastava <srivastava.amrit1999@gmail.com>

Co-authored-by: Rajani169 <sharmarajani169@gmail.com>
Co-authored-by: ksvkabra <35252448+ksvkabra@users.noreply.github.com>

Co-authored-by: Amrit Srivastava <srivastava.amrit1999@gmail.com>
Co-authored-by: Rajani169 <sharmarajani169@gmail.com>

* Keshavk- added blog section in the home page (#69)

* added blog section in the home page

* Styling changes in the blog card

* Styling fixes and added blogs data

* added link to the read more button

* Added id in the index for blogs

* Update Committees Data (#72)

* feat: Modify Committees Page

* Modified Committees Data

* update: committees data

Co-authored-by: Rajani169 <sharmarajani169@gmail.com>

* wip: update slides through state

Co-authored-by: Rajani169 <sharmarajani169@gmail.com>
Co-authored-by: Aditya Agarwal <aditya81070@gmail.com>
Co-authored-by: jecrcmun19 <41872688+jecrcmun19@users.noreply.github.com>
Co-authored-by: ksvkabra <35252448+ksvkabra@users.noreply.github.com>

amritsrivastava commented:
closing this PR with the reference of #78
